en/pe-docs/docs/choreo-concepts/choreo-marketplace.md

```diff
@@ -0,0 +1,76 @@
+# Choreo Marketplace
+
+The Choreo Marketplace promotes and facilitates reusing and sharing services. It allows you to share all the services deployed in Choreo.
```

Comment:
https://github.com/wso2-enterprise/choreo/issues/34149 Created this issue. Anyway I am +1 to keep this atm.